app/controllers/api/jobs_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module Api
+  class JobsController < ApplicationController
+    def index
+      jobs = Job.all
+      render status: 200, json: { data: jobs.as_json }
```

Comment:
This controller action returns all jobs on the jobs board. When we no longer want a job on the board, we plan to delete the record entirely.